### PR TITLE
NSM CSI driver (continuation)

### PR DESCRIPTION
**Pull request by glazychev-art**
## Description
Continuation of https://github.com/networkservicemesh/cmd-csi-driver/pull/1
Closes: https://github.com/networkservicemesh/cmd-csi-driver/pull/1
Issue: https://github.com/networkservicemesh/deployments-k8s/issues/9218

This PR adds NSM CSI driver implementation, that is based on [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi) with a few changes for NSM.
`github.com/spiffe/spiffe-csi/pkg/mount` was fully reused, because no changes were made to it.

### CODE REVIEW

**Comment by glazychev-art** [comment]
@szvincze 
Could you please take a look?

File: main.go [hunk 1]
```diff
@@ -18,5 +16,73 @@
...
+	"github.com/networkservicemesh/cmd-csi-driver/pkg/driver"
+	"github.com/networkservicemesh/cmd-csi-driver/pkg/logkeys"
+	"github.com/networkservicemesh/cmd-csi-driver/pkg/server"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+)
+
+var (
+	pluginNameFlag    = flag.String("plugin-name", "csi.networkservicemesh.io", "Plugin name to register")
+	nsmSocketDirFlag  = flag.String("nsm-socket-dir", "", "Path to the NSM API socket directory")
+	csiSocketPathFlag = flag.String("csi-socket-path", "/nsm-csi/csi.sock", "Path to the CSI socket")
```
Comment (denis-tingaikin) [inline main.go:37]: Could be moved to env config

**Comment by szvincze** [comment]
> @szvincze Could you please take a look?

I checked and looks good to me.

File: main.go (hunk 1 above)
Comment (glazychev-art) [reply main.go]: Fixed

**Pull request merged**